rebar.config

```diff
@@ -45,11 +43,10 @@
   {jiffy, ".*", {git, "https://github.com/davisp/jiffy.git", "0.14.11"}},
   {fusco, ".*", {git, "https://github.com/esl/fusco.git", {branch, "master"}}},
   {idna, ".*", {git, "https://github.com/benoitc/erlang-idna.git", {tag, "1.2.0"}}},
-  {poolboy, ".*", {git, "https://github.com/devinus/poolboy.git", {tag, "1.5.1"}}},
   {uuid, ".*", {git, "https://github.com/okeuday/uuid.git", {tag, "v1.7.1"}}},
   {fast_tls, ".*", {git, "https://github.com/processone/fast_tls.git", "a166f0e9fe78304e5ca628fd5eff57c850241813"}},
   {lasse, ".*", {git, "https://github.com/inaka/lasse.git", "692eaec"}},
-  {worker_pool, ".*", {git, "https://github.com/inaka/worker_pool.git", {tag, "3.0.0"}}},
+  {worker_pool, ".*", {git, "https://github.com/inaka/worker_pool.git", {branch, "fix-spec-3.0.0"}}},
```

Comment:
I'd like to see a PR from this branch of worker_pool. I remember merging inaka/worker_pool#139 I understand this was not enough. When this new PR is merged we can change the worker_pool ref to the merge commit in master branch.

Reply:
There is more to it - using master from worker_pool crashed everything, dunno why, something important must have changed. So I branched off from the release. Plus, yes, there was one more place where a spec needed fixing, and I spotted even more such places, but they didnt crash mongoose. I'll try to find some time to clean it all up, but it is a worker_pool related task, not mongooseim.

Reply:
That's bad, we better improve this situation soon. I'm not saying in this PR, but preferably before next release.

Reply:
Yes, definitely. I hope to be able some day to do some work on worker_pool, spec cleanup included, but mostly the thingy we discussed before.